mlir/include/mlir/Dialect/Polynomial/IR/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(LLVM_TARGET_DEFINITIONS PolynomialOps.td)
+mlir_tablegen(PolynomialDialect.cpp.inc -gen-dialect-defs -dialect=polynomial)
+mlir_tablegen(PolynomialDialect.h.inc -gen-dialect-decls -dialect=polynomial)
+add_public_tablegen_target(MLIRPolynomialDialectIncGen)
```

Comment:
`add_dependencies(mlir-headers MLIRPolynomialDialectIncGen)` may be needed here.

Reply:
Fixed in 4f8be047edc